packages/components/src/buttons.tsx

```diff
@@ -10,17 +10,17 @@ const buttonStyle: CSSProperties = {
   height: '2.25rem',
   fontSize: 14,
   fontWeight: 500,
-  textTransform: 'uppercase',
   whiteSpace: 'nowrap',
   userSelect: 'none',
 };
 
 export interface ButtonPrimaryProps extends ClickableProps {
   danger?: boolean;
+  inheritText?: boolean;
 }
 
 export const ButtonPrimary = (props: ButtonPrimaryProps): ReactNode => {
-  const { children, danger = false, disabled, hover, style, ...otherProps } = props;
+  const { children, danger = false, inheritText = false, disabled, hover, style, ...otherProps } = props;
```

Comment:
You can just pass style information to ButtonPrimary, and it will override the default styling. I think that is preferable to adding individual arguments.

Reply:
Wonderful, thank you!